paasta_tools/cli/cmds/spark_run.py

```diff
@@ -395,6 +395,18 @@ def add_subparser(subparsers):
         default=DEFAULT_AWS_REGION,
     )
 
+    aws_group.add_argument(
+        "--assume-aws-role",
+        help="Takes an ARN and will attempt to create a session",
```

Comment:
thoughts on expanding on this a tad with something like
```suggestion
        help="Takes an AWS IAM role ARN and attempts to create a session",
```